README.md

```diff
@@ -87,7 +87,7 @@ bundle exec evt-pg-create-db
 
 Form more background on the Postgres message store database, you can explore the SQL scripts at:
 
-https://github.com/eventide-project/message-store-postgres/tree/master/database
+https://github.com/eventide-project/message-store-postgres/tree/master/scripts
```

Comment:
Actually, the `scripts` directory has runners for the SQL scripts, but it's the `database` folder that has the SQL files that define the database. I think I can clarify the wording so that it's less ambiguous.